mne/channels/montage.py

```diff
@@ -483,11 +487,6 @@ def read_dig_montage(hsp=None, hpi=None, elp=None, point_names=None,
 
     .. versionadded:: 0.9.0
     """
-    if not isinstance(unit, string_types) or unit not in('m', 'mm', 'cm'):
-        raise ValueError('unit must be "m", "mm", or "cm"')
```

Comment:
Thanks for catching that! See https://github.com/mne-tools/mne-python/pull/3320/commits/76ab987ba6aa498fd39742888264f0bc0558359f